### PR TITLE
adding BlackBoxAuditing

### PR DESCRIPTION
**Pull request by pavelgrib**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/blackboxauditing/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - networkx
+    - scikit-learn
+
+test:
+  imports:
+    - BlackBoxAuditing
+
+about:
+  home: http://github.com/algofairness/BlackBoxAuditing
+  license: Apache 2.0
```
Comment (marcelotrevisani) [inline recipes/blackboxauditing/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/blackboxauditing/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 02824682f4ac3a89e606d3dc7e99481d09e82655513511e9406268f8b17be3d6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/blackboxauditing/meta.yaml]: I think this recipe could be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/blackboxauditing/meta.yaml (hunk 1 above)
Comment (pavelgrib) [reply recipes/blackboxauditing/meta.yaml]: made the change, thanks

File: recipes/blackboxauditing/meta.yaml (hunk 2 above)
Comment (pavelgrib) [reply recipes/blackboxauditing/meta.yaml]: adding `noarch: python` under the build section is the way to do this right?

**Comment by pavelgrib** [comment]
@marcelotrevisani thanks for taking a look.  checks are passing i skipped "win" because of some  specific issues with that build.

File: recipes/blackboxauditing/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: 02824682f4ac3a89e606d3dc7e99481d09e82655513511e9406268f8b17be3d6
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python 3
```
Comment (scopatz) [inline recipes/blackboxauditing/meta.yaml]: Should be `python >=3` here andf below

File: recipes/blackboxauditing/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/blackboxauditing/meta.yaml]: Yep!

File: recipes/blackboxauditing/meta.yaml (hunk 3 above)
Comment (pavelgrib) [reply recipes/blackboxauditing/meta.yaml]: @scopatz fixed, thanks!

**Pull request merged**